src/webviews/apps/plus/home/components/active-work.ts

```diff
@@ -75,14 +76,14 @@ export class GlActiveWork extends SignalWatcher(LitElement) {
 
 	private renderComplete(overview: Overview) {
 		const activeBranches = overview?.repository?.branches?.active;
-		if (activeBranches == null) return html`<span>None</span>`;
+		if (!activeBranches) return html`<span>None</span>`;
 
 		return html`
 			<h3 class="section-heading section-heading--actions">
 				<span>Active (${activeBranches.length})</span>
-				${when(overview!.choices, () => html`<gl-change-repo .options=${overview!.choices}></gl-change-repo>`)}
+				${when(overview.choices, () => html`<gl-change-repo .options=${overview.choices}></gl-change-repo>`)}
 			</h3>
-			${activeBranches.map(branch => this.renderRepoBranchCard(overview!.repository.name, branch))}
+			${repeat(activeBranches, branch => this.renderRepoBranchCard(overview.repository.name, branch))}
```

Comment:
`repeat` isn't really beneficial without adding a `keyFn`

```suggestion
			${repeat(activeBranches, branch => branch.name,  branch => this.renderRepoBranchCard(overview.repository.name, branch))}
```